src/MSIdentityScaffolding/Microsoft.DotNet.MSIdentity/Tool/MsAADTool.cs

```diff
@@ -67,42 +67,58 @@ internal async Task<string> PrintApplicationsList()
             var graphObjects = await GraphServiceClient.Me.OwnedObjects
                             .Request()
                             .GetAsync();
-            IList<Application> applicationList = new List<Application>();
-            if (graphObjects != null && graphObjects.Any())
+
+            if (graphObjects != null)
```

Comment:
Seems like these two algorithms are slightly different on "T" and on the way to print them out after the querying is done.  Factor together the "T" parts, and then have two separate methods that can print them?

Reply:
done.